src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/controllers/advice/CallbackControllerAdvice.java

```diff
@@ -1,42 +0,0 @@
-package uk.gov.hmcts.reform.iacasenotificationsapi.infrastructure.controllers.advice;
-
-import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
-
-import javax.servlet.http.HttpServletRequest;
-import lombok.extern.slf4j.Slf4j;
-import org.apache.commons.lang3.exception.ExceptionUtils;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.ControllerAdvice;
-import org.springframework.web.bind.annotation.ExceptionHandler;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.context.request.RequestAttributes;
-import org.springframework.web.context.request.RequestContextHolder;
-import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
-import uk.gov.hmcts.reform.iacasenotificationsapi.domain.RequiredFieldMissingException;
-
-@Slf4j
-@ControllerAdvice(basePackages = "uk.gov.hmcts.reform.iacasenotificationsapi.infrastructure.controllers")
```

Comment:
We need to check this isnt being created as a spring bean and being added to the spring context actually - since it might not be explicitly used but RequestBodyAdviceAdapter is a @Component so will be added to the spring context, and now sure if it will be used by any background spring processes